WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -84,7 +85,11 @@ class NotificationDetailsViewController: UIViewController, NoResultsViewHost {
     ///
     weak var dataSource: NotificationsNavigationDataSource?
 
-    /// Notification to-be-displayed
+    /// Used to present CommentDetailViewController when previous/next notification is a Comment.
+    ///
+    var notificationCommentDetailCoordinator: NotificationCommentDetailCoordinator?
```

Comment:
Maybe make this a `weak var` just to be safe? The `completion` closure in `createViewController` is not retained in the coordinator, but it is still an escaping parameter.
```suggestion
    weak var notificationCommentDetailCoordinator: NotificationCommentDetailCoordinator?
```